cmd/scanGithub.go

```diff
@@ -79,16 +83,20 @@ func init() {
 	scanGithubCmd.Flags().String("bind-address", "127.0.0.1", "The IP address for the webserver")
 	scanGithubCmd.Flags().Int("bind-port", 9393, "The port for the webserver")
 	scanGithubCmd.Flags().Int("commit-depth", 0, "Set the depth for commits")
+	scanGithubCmd.Flags().Bool("csv", false, "Write results to --output-file in CSV format")
 	scanGithubCmd.Flags().Bool("debug", false, "Print debugging information")
 	scanGithubCmd.Flags().String("github-api-token", "", "API token for access to github, see doc for necessary scope")
 	scanGithubCmd.Flags().String("github-targets", "", "A space separated list of github.com users or orgs to scan")
 	scanGithubCmd.Flags().Bool("hide-secrets", false, "Hide secrets from output")
 	scanGithubCmd.Flags().String("ignore-extension", "", "a comma separated list of extensions to ignore")
 	scanGithubCmd.Flags().String("ignore-path", "", "a comma separated list of paths to ignore")
 	scanGithubCmd.Flags().Bool("in-mem-clone", false, "Clone repos in memory")
+	scanGithubCmd.Flags().Bool("json", false, "Write results to --output-file in JSON format")
 	scanGithubCmd.Flags().Int("match-level", 3, "Signature match level")
 	scanGithubCmd.Flags().Int("max-file-size", 50, "Max file size to scan")
 	scanGithubCmd.Flags().Int("num-threads", 0, "The number of threads to execute with")
+	scanGithubCmd.Flags().String("output-dir", "./", "Write csv and/or json files to directory")
```

Comment:
this needs to be changed to *~/.wraith* or *$HOME/.wraith* and confirmed the paths are expanded correctly. The issue here being if the run the binary in */usr/bin* they may not have the permissions to create an output file. We can install it into */usr/local/bin* if we want but that does not mean the user won't move it or compile it themselves and put it in */usr/bin* with everything else.

For example

<img width="571" alt="Screen Shot 2020-10-14 at 10 44 27 PM" src="https://user-images.githubusercontent.com/672940/96070788-deacf380-0e6e-11eb-84b3-f1c6d0d8c0bd.png">